test/attachto.js

```diff
@@ -33,9 +34,9 @@ describe('attachTo', function() {
         attachTo(elm, h('div#attached', 'New text')),
       ]),
     ]);
-    patch(vnode0, vnode1);
+    elm = patch(vnode0, vnode1).elm;
```

Comment:
Not sure if it's necessary for this specific test, but it is for most tests since some root nodes are mutated. At first, I modified all the tests then I changed my mind, maybe this one was left out.

But maybe your remark is for all the tests?